src/rat/run_rat.py

```diff
@@ -88,6 +90,11 @@ def run_rat(config_fn, operational_latency=None):
             config_copy = copy.deepcopy(config)
             # Run RAT for basin
             no_errors, latest_altimetry_cycle = rat_basin(config, log)
+            # Run RAT forecast for basin if forecast is True
+            if config['PLUGINS']['forecast']:
+                log.info('############## Starting RAT forecast for '+config['BASIN']['basin_name']+' #################')
+                no_errors = forecast(config, log)
+                log.info('############## RAT forecast finished for '+config['BASIN']['basin_name']+ ' with '+str(no_errors)+' error(s). #################')
```

Comment:
Don't use no_errors again. Instead we can use forecast_no_errors just to distinguish between the two. no_errors is later being used below.

Reply:
Thanks, replaced no_errors with forecast_no_errors. Also added a log statement to record the number of errors.